_sub/compute/eks-addons/dependencies.tf

```diff
@@ -23,7 +23,7 @@ locals {
     "1.21" = "v1.8.4-eksbuild.1"
     "1.22" = "v1.8.7-eksbuild.1"
     "1.23" = "v1.8.7-eksbuild.3"
-    "1.24" = "v1.8.7-eksbuild.3"
+    "1.24" = "v1.9.3-eksbuild.2"
```

Comment:
This should be v1.8.7-eksbuild.4, because this is the recommended version of coredns managed add-on for 1.24